<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="horizontalFrame">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="ParameterBtn">
         <property name="text">
          <string>图像参数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="EnhanceBtn">
         <property name="text">
          <string>图像增强</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="TransformBtn">
         <property name="text">
          <string>图像变换</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GrainBtn">
         <property name="text">
          <string>图像加噪/模糊</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CodingBtn">
         <property name="text">
          <string>图像编码</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="OpenImage"/>
    <addaction name="SaveImage"/>
    <addaction name="FreshImage"/>
    <addaction name="separator"/>
    <addaction name="ActionExit"/>
   </widget>
   <widget class="QMenu" name="AboutMenu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="ActionInfo"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="AboutMenu"/>
  </widget>
  <action name="OpenImage">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="SaveImage">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="ActionExit">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="ActionDemo">
   <property name="text">
    <string>实验说明</string>
   </property>
  </action>
  <action name="ActionInfo">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="Actionhomepage">
   <property name="text">
    <string>课程主页</string>
   </property>
  </action>
  <action name="FreshImage">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
